FirebaseAdmin/FirebaseAdmin/Auth/IUserInfo.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FirebaseAdmin.Auth
+{
+    /// <summary>
+    /// A collection of standard profile information for a user. Used to expose profile information
+    /// returned by an identity provider.
+    /// </summary>
+    public interface IUserInfo
+    {
+        /// <summary>
+        /// Returns the user's unique ID assigned by the identity provider.
+        /// </summary>
+        /// <returns>a user ID string.</returns>
+        string GetUid();
```

Comment:
I think all these should be properties instead of methods.

Reply:
I can do that, but wouldn't we be departing from the Java implementation if we renamed it to Uid (I would imagine the "Get" naming convention wouldn't make sense for a property)?

Reply:
Properties are more idiomatic in C#. So we should have something like this:

```
string Uid { get; }
string Email { get; }
// and so on...
```